tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,78 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def adding_label(instance):
```

Comment:
More appropriate defination name would be `_add_host_label`

Reply:
Thanks @vamahaja
Updated